### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Abidoyinsola1**
A clone of The New York Times Article Page.

### CODE REVIEW

**Comment by tirthajyoti-ghosh** [comment]
# Pull Request under review by T. Ghosh (TSE)

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Impressive project team ✨

But there are couple of issues that still needed to be addressed before you can merge. Check the comments below for more details.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,856 @@
...
+    />
+  </head>
+  <body>
+    <nav>
+      <ul>
+        <li><i class="fa fa-bars" aria-hidden="true"></i></li>
+        <li><i class="fa fa-search" aria-hidden="true"></i></li>
+        <li><a href="#">SPACE &amp; COSMOS</a></li>
+      </ul>
+      <img src="./assets/images/NewYork.PNG" alt="The New YorK Times" />
```
Comment (tirthajyoti-ghosh) [inline index.html]: - The NY Times logo must be a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

File: index.html [hunk 2]
```diff
@@ -0,0 +1,856 @@
...
+      <div class="nav-right">
+        <a href="#">Subscribe now</a>
+        <button>Login</button>
+      </div>
+    </nav>
```
Comment (tirthajyoti-ghosh) [inline index.html]: - The `nav` element shrinks when the height is changed.
Annotation 2020-06-09 103602

File: index.html [hunk 3]
```diff
@@ -0,0 +1,856 @@
...
+      <ul>
+        <li><i class="fa fa-bars" aria-hidden="true"></i></li>
+        <li><i class="fa fa-search" aria-hidden="true"></i></li>
+        <li><a href="#">SPACE &amp; COSMOS</a></li>
+      </ul>
+      <img src="./assets/images/NewYork.PNG" alt="The New YorK Times" />
+      <div class="nav-right">
+        <a href="#">Subscribe now</a>
+        <button>Login</button>
+      </div>
```
Comment (tirthajyoti-ghosh) [inline index.html]: - The styles are different than the original.
  Original 👇  
  Annotation 2020-06-09 222448

  Right now 👇  
  Annotation 2020-06-09 222437

File: index.html [hunk 4]
```diff
@@ -0,0 +1,856 @@
...
+            <li>
+              <a href=""><i class="fa fa-bookmark"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fa fa-comment"></i></a>
+            </li>
+          </ul>
+        </div>
+        <div class="container">
+          <div class="main-paragraph">
```
Comment (tirthajyoti-ghosh) [inline index.html]: - The `main-paragraph` must be centered.
  
  Original 👇  
  Annotation 2020-06-09 222838

  Right now 👇  
  Annotation 2020-06-09 222810

File: index.html [hunk 5]
```diff
@@ -0,0 +1,856 @@
...
+            </p>
+            <p>
+              Dr. Kovac and his collaborators, working in an experiment known as
+              Bicep, for Background Imaging of Cosmic Extragalactic
+              Polarization, reported their results in a scientific briefing at
+              the Center for Astrophysics here on Monday and in a set of papers
+              submitted to The Astrophysical Journal.
+            </p>
+          </div>
+          <aside>
```
Comment (tirthajyoti-ghosh) [inline index.html:181]: - Reduce the size of images and texts to match the original and shift the `aside` tag to the right.
  
  Original 👇
  
  Annotation 2020-06-09 223329

  Right now 👇
  
  Annotation 2020-06-09 223308

File: index.html [hunk 6]
```diff
@@ -0,0 +1,856 @@
...
+            <a href="#">Order Reprints</a> | <a href="#">Today’s Paper</a> |
+            <a href="#">Subscribe</a>
+          </p>
+          <button>Read 615 Comments</button>
+          <ul>
+            <li><i class="fa fa-facebook"></i></li>
+            <li><i class="fa fa-twitter"></i></li>
+            <li><i class="fa fa-envelope"></i></li>
+            <li><i class="fa fa-share"></i></li>
+          </ul>
```
Comment (tirthajyoti-ghosh) [inline index.html]: - The icons must be links.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,856 @@
...
+                  <figcaption><small>NASA</small></figcaption>
+                </figure>
+                <h3 class="article-title">
+                  NASA Names Dark Energy Telescope for Nancy Grace Roman
+                </h3>
+                <p>May 20</p>
+              </div>
+            </div>
+          </div>
+          <div class="rec-editor-pick">
```
Comment (tirthajyoti-ghosh) [inline index.html:629]: - Add some space below this `rec-editor-picks`. Right now it's sticking to the `footer`.  
  
  Annotation 2020-06-09 224045

File: index.html [hunk 8]
```diff
@@ -0,0 +1,856 @@
...
+          <li><a>Site Map</a></li>
+          <li><a>Help</a></li>
+          <li><a>Subscriptions</a></li>
+        </ul>
+      </div>
+    </footer>
+
+    <div class="footer-nav">
+      <p>Up-to-date information on coronavirus is available for free</p>
+      <div class="expand">
```
Comment (tirthajyoti-ghosh) [inline index.html:863]: - Align `expand` with the other elements. Right now it has extra space on top.
  
  Annotation 2020-06-09 224142

**Review by tirthajyoti-ghosh — COMMENTED** [review]
[OPTIONAL] Decrease the font size of `section.hero h1`.

*Since you are using Github actions, you can disable stickler on this project.*

Apply these changes and request for another code review.

Happy coding 👨‍💻

File: index.html [hunk 9]
```diff
@@ -0,0 +1,869 @@
...
+            <li>
+              <a href="#!"><i class="fa fa-share"></i></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="recommendation-section">
+        <div class="recommendation-area">
+          <div class="more-in-space">
+            <hr style="margin: 50px 0 0;" />
```
Comment (HADDADSOHAIB) [inline index.html]: Please remove inline styles

File: index.html [hunk 10]
```diff
@@ -0,0 +1,869 @@
...
+                </figure>
+                <h3 class="article-title">
+                  Wallace Stegner and the Conflicted Soul of the West
+                </h3>
+                <p class="date">June 3</p>
+              </div>
+            </div>
+          </div>
+          <div class="most-popular">
+            <hr style="margin: 50px 0 0;" />
```
Comment (HADDADSOHAIB) [inline index.html]: Please remove inline styles

File: index.html [hunk 11]
```diff
@@ -0,0 +1,869 @@
...
+        <li><a href="#">SPACE &amp; COSMOS</a></li>
+      </ul>
+      <a href="#">
+        <img src="./assets/images/NewYork.PNG" alt="The New YorK Times" />
+      </a>
+      <div class="nav-right">
+        <button>Subscribe now</button>
+        <button>Login</button>
+      </div>
+    </nav>
```
Comment (HADDADSOHAIB) [inline index.html:29]: As requested by the previous TSE, the nav shrinks when you reduce the height, please fix that.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,869 @@
...
+                <li>
+                  <a href=""><i class="fa fa-share social"></i></a>
+                </li>
+                <li>
+                  <a href=""><i class="fa fa-bookmark"></i></a>
+                </li>
+                <li>
+                  <a href=""><i class="fa fa-comment"></i></a>
+                </li>
+              </ul>
```
Comment (HADDADSOHAIB) [inline index.html]: These icons should be links, add href="#" so they will look as link (the cursor will have a pointer)

File: index.html [hunk 13]
```diff
@@ -0,0 +1,869 @@
...
+                    >
+                  </figcaption>
+                </figure>
+                <h3 class="article-title">
+                  Wallace Stegner and the Conflicted Soul of the West
+                </h3>
+                <p class="date">June 3</p>
+              </div>
+            </div>
+          </div>
```
Comment (HADDADSOHAIB) [inline index.html]: All these articles (the articles with images in the footer) should be links:
Screenshot from 2020-06-10 13-32-44

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @Abidoyinsola1 and @somoye123     :handshake: 

### Good job so far :clap:  your project looks like the original :medal_sports: 

### there are some changes to get the project approved :point_down: , please check the comments and once you are done with the requirements ask for another code review.

File: index.html (hunk 11 above)
Comment (Abidoyinsola1) [reply index.html:29]: Hello @haddadsohaib can you explain more on what you mean by the reduced height? A screenshot would help in clarifying. Thank you.

**Comment by Abidoyinsola1** [comment]
All necessary corrections have been made, but more explanation is needed on the issue raised concerning the navigation bar.

**Comment by HADDADSOHAIB** [comment]
- Check the differences between these two:
- Original article:
Screenshot from 2020-06-10 14-51-15
- Your project:
Screenshot from 2020-06-10 14-51-33
When the height of the page is reduced, the nav become smaller and the logo, buttons don't fit any more, in the original article the nav height is a fixed value

File: index.html (hunk 11 above)
Comment (HADDADSOHAIB) [reply index.html:29]: @Abidoyinsola1  hi, check my comment down I added some screenshots

**Comment by somoye123** [comment]
Nav shrinking issues when the height is reduced as been fixed

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi **TEAM**, you guys did a great job fixing all the comments pointed out by the previous TSE's :+1: Thanks to your persistent desire to improve and the guidance of our TSE's :slightly_smiling_face: Our project now meets all the minimum requirements :hugs: 

You can now merge to master and submit the project completion form
Good luck with your next projects :100: 
_Happy coding_ :woman_technologist: :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**